crypto/ec/ecp_nistp224.c

```diff
@@ -350,7 +348,7 @@ static int BN_to_felem(felem out, const BIGNUM *bn)
         ECerr(EC_F_BN_TO_FELEM, EC_R_BIGNUM_OUT_OF_RANGE);
         return 0;
     }
-    num_bytes = BN_bn2bin(bn, b_in);
+    num_bytes = BN_bn2binpad(bn, b_in, sizeof(b_in));
     flip_endian(b_out, b_in, num_bytes);
```

Comment:
could you use BN_lebin2bn, instead of flip_endian which is not constant time at all?

Reply:
I mean BN_lebin2bnpad of course.

Reply:
In what way is `flip_endian` not constant-time? It's just looping over `num_bytes`, which is just the public byte count. (Although yeah if you have a little-endian-specific function, may as well use it.)

Reply:
Aehm, actually BN_bn2lebinpad :-)  maybe that needs some rework first,
to be more constant time like bn2binpad does?

Reply:
Ah, good point, I thought num_bytes is BN_num_bytes...

Reply:
Good idea, I removed `flip_endian()` completely from the three 128-bit implementations, replacing with `BN_lebin2bn()` and `BN_bn2lebinpad()`.

I also unified the codepath of `BN_bn2lebinpad()` and `BN_bn2binpad()` so they have exactly the same amount of countermeasures against SCA.